llvm/llvm.spec.in

```diff
@@ -5,21 +5,20 @@ Summary: LLVM (An Optimizing Compiler Infrastructure)
 License: Apache-2.0 with LLVM exception
 Vendor: None (open source)
 Group: Development/Compilers
-URL: http://llvm..org/
-Source: http://llvm.org/releases/@PACKAGE_VERSION@/@PACKAGE_NAME@-@PACKAGE_VERSION@.tar.gz
+URL: https://llvm.org/
+Source: https://llvm.org/releases/@PACKAGE_VERSION@/@PACKAGE_NAME@-@PACKAGE_VERSION@.tar.gz
 BuildRoot: %{_tmppath}/%{name}-root
 Requires: /sbin/ldconfig
-BuildRequires: gcc >= 3.4
+BuildRequires: gcc >= 7.4
 
 %description
 LLVM is a compiler infrastructure designed for compile-time, link-time, runtime,
 and idle-time optimization of programs from arbitrary programming languages.
 LLVM is written in C++ and has been developed since 2000 at the University of
-Illinois and Apple. It currently supports compilation of C and C++ programs,
-using front-ends derived from GCC 4.0.1. A new front-end for the C family of
-languages is in development. The compiler infrastructure
-includes mirror sets of programming tools as well as libraries with equivalent
-functionality.
+Illinois and Apple. C-like languages use the Clang front end. This component
```

Comment:
Not sure if this `C-like` sounds right. I'd rather say `The C language and its derivatives` (or successors), well I'm not English native, I can't definitely say what could possibly sound better.

Reply:
I copied this sentence from https://llvm.org/docs/GettingStarted.html#getting-the-source-code-and-building-llvm, and it should be fine to say it like this.